packages/core/src/components/forms/formGroup.tsx

```diff
@@ -72,22 +82,13 @@ export class FormGroup extends React.PureComponent<IFormGroupProps, {}> {
         );
     }
 
-    private getClassName() {
-        const { className, disabled, inline, intent } = this.props;
-        return classNames(
-            Classes.FORM_GROUP,
-            Classes.intentClass(intent),
-            {
-                [Classes.DISABLED]: disabled,
-                [Classes.INLINE]: inline,
-            },
-            className,
-        );
-    }
-
     private maybeRenderRequiredLabel() {
         const { requiredLabel } = this.props;
-        return requiredLabel === true ? FormGroup.DEFAULT_REQUIRED_CONTENT : requiredLabel;
+        if (requiredLabel !== false) {
+            const label = requiredLabel === true ? FormGroup.DEFAULT_REQUIRED_CONTENT : requiredLabel;
+            return <span className={Classes.TEXT_MUTED}> {label}</span>;
```

Comment:
this is not the correct place to add the space, and it's causing the tests to fail. please remove.